crates/virtual-dom-rs/tests/diff_patch_test_case/mod.rs

```diff
@@ -5,9 +5,13 @@ use virtual_dom_rs::prelude::*;
 
 /// A test case that both diffing and patching are working in a real browser
 pub struct DiffPatchTest<'a> {
+    /// Description of the test case.
     pub desc: &'static str,
+    /// The old virtual node.
     pub old: VirtualNode,
+    /// The new virtual node.
     pub new: VirtualNode,
+    /// TODO: What does this do?
```

Comment:
By default we generate the expected based on `new.to_string()`

https://github.com/chinedufn/percy/blob/7934e8b10cc91b109bf732ba2a5d463d939f1510/crates/virtual-dom-rs/tests/diff_patch_test_case/mod.rs#L37

This ability to override that and instead use our own string is used here

https://github.com/chinedufn/percy/blob/7934e8b10cc91b109bf732ba2a5d463d939f1510/crates/virtual-dom-rs/tests/diff_patch.rs#L96-L100

We use it to account for the `Percy Text Node Separator` (`<!--ptns -->` ) comment that we insert into the DOM in between virtual text nodes that we're adding into the browser in order to prevent them from getting merged by the browser.

Reply:
Thanks, I added a doc comment.